cryptlex/cryptlex-enterprise/values.yaml

```diff
@@ -180,6 +180,20 @@ webApi:
     rateLimit: 50
     ipWhitelist: []
     realIpHeader:
+  # Pod security context
+  podSecurityContext:
```

Comment:
@mumin-khan this place usually takes app specific config, this can go service specif config at top